src/AdminAcceptanceTests.Actions/Utils/EmailServerDriver.cs

```diff
@@ -18,14 +19,14 @@ public static async Task<int> GetEmailCountAsync(string hostUrl)
         public static async Task<IEnumerable<Email>> FindAllEmailsAsync(string hostUrl)
         {
             HttpResponseMessage response = new HttpResponseMessage();
-
-            using var client = new HttpClient();
-            response = await client.GetAsync($"{hostUrl}:1080/email");
+            using var client = NewHttpClient();
+            response = await client.GetAsync(GetAllEmailsUrl(hostUrl));
```

Comment:
Can you define the variable here rather than separately defined?

Reply:
What do you mean?

Reply:
Rather than defining response above the using statement, have `var response = await...`